dali/test/python/test_autoserialize.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Note that this file probably doesn't match any pattern form the qa/ scripts. Naming it test_pipeline_autoserialize would do. Or adding it explicitly.

Reply:
Added